runtime/src/main/java/org/corfudb/protocols/CorfuProtocolLogData.java

```diff
@@ -29,7 +29,7 @@ public static LogDataMsg getLogDataMsg(LogData logData) {
         logData.doSerialize(buf);
 
         return LogDataMsg.newBuilder()
-                .setEntry(ByteString.copyFrom(buf.resetReaderIndex().array()))
+                .setEntry(ByteString.copyFrom(buf.array(), buf.readerIndex(), buf.writerIndex()))
                 .build();
     }
```

Comment:
Do we want to copy from readerIndex to writerIndex? I thought the third argument of `copyFrom` is the number of bytes to copy.

Reply:
Fixed, Thanks!